src/components/Layout/MobileNavBar.tsx

```
@@ -1,64 +1,86 @@
 import { useAtom, useSetAtom } from 'jotai';
-import { Map, CheckSquare, Filter, Calendar, Lightbulb, AlertTriangle } from 'lucide-react';
+import { Map, Calendar, Sparkles, MoreHorizontal } from 'lucide-react';
```

Biome (2.1.2)

[error] Do not shadow the global "Map" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)